conformance_tests/core/test_cmdqueue/src/test_cmdqueue.cpp

```diff
@@ -235,6 +235,155 @@ TEST_P(
   }
 }
 
+class zeImmediateCommandListAppendCommandListsExpTests
+    : public ::testing::Test,
+      public ::testing::WithParamInterface<CustomExecuteParams> {
+protected:
+  void SetUp() override {
+    ze_device_handle_t device = lzt::zeDevice::get_instance()->get_device();
+    const ze_driver_handle_t driver = lzt::get_default_driver();
+    const ze_context_handle_t context = lzt::get_default_context();
+    EXPECT_GT(params.num_command_lists, 0);
+
+    print_cmdqueue_exec(params.num_command_lists, params.sync_timeout);
+
+    ze_command_queue_desc_t cmdQueueDesc = {
+        ZE_STRUCTURE_TYPE_COMMAND_QUEUE_DESC};
+    ze_command_list_desc_t cmdListDesc = {ZE_STRUCTURE_TYPE_COMMAND_LIST_DESC};
+
+    cmdQueueDesc.ordinal = lzt::getComputeQueueGroupOrdinal(device);
+
+    cmdQueueDesc.index = 0;
+    cmdQueueDesc.mode = ZE_COMMAND_QUEUE_MODE_ASYNCHRONOUS;
+
+    EXPECT_EQ(ZE_RESULT_SUCCESS,
+              zeCommandListCreateImmediate(context, device, &cmdQueueDesc,
```

Comment:
We should prefer `lzt::create_immediate_command_list`, the same goes for many other L0 API invocations below

Reply:
done!